src/app/types/UserType.tsx

```diff
@@ -0,0 +1,20 @@
+export interface User {
+    isAdmin: boolean;
+    firstName: string;
+    lastName: string;
+    middleName?: string;
+    address: string;
+    school: string;
+    birthdate: string;
```

Comment:
Typescript has a builtin Date type, so to make age calculations easier, let's have birthdate as a Date type